functions-methods.js

```diff
@@ -33,4 +51,16 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity() {
+    const email3 = "tessmellink@novi,nl"
+    if (email3.includes("@") && !email3.includes(",") && !email3.endsWith(".")) {
+        return "True";
+    } else {
+        return "False";
+    }
```

Comment:
Dit kun je nog wat inkorten
```suggestion
    return email3.includes("@") && !email3.includes(",") && !email3.endsWith(".");
```